src/main/java/ua/antibyte/analyzer/entity/Role.java

```diff
@@ -0,0 +1,38 @@
+package ua.antibyte.analyzer.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "roles")
+@NoArgsConstructor
+@Data
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "role_name")
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public Role(RoleName roleName) {
```

Comment:
```suggestion
    private Role(RoleName roleName) {
```
is it supposed to be used only by `of` method?